### PR TITLE
Categories CRUD

### PR DESCRIPTION
**Pull request by System-Glitch**
# Merge branchCategoriesCRUD

## Description

Added a CRUD for Categories.

### Possible drawbacks

None.

## Related issue(s)

List all related issues here:

* #13 (Controllers and routes)

## Additional information

Authorizations have to be implemented too.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,61 +1,58 @@
 {
-    "name": "laravel/laravel",
```
Comment (TheElectronWill) [inline composer.json]: Is there any meaningful change besides indentation?

File: database/migrations/2018_08_07_184155_create_threads_and_comments.php [hunk 2]
```diff
@@ -53,7 +53,7 @@ public function up()
         Schema::create('categories', function (Blueprint $table) {
             $table->softDeletes();
             $table->increments('id');
-            $table->unsignedInteger('parent_id');
+            $table->unsignedInteger('parent_id')->nullable();
```
Comment (TheElectronWill) [inline database/migrations/2018_08_07_184155_create_threads_and_comments.php]: good catch :+1:

File: composer.json (hunk 1 above)
Comment (System-Glitch) [reply composer.json]: The `helper.php` file has been added to the autoloaded files.

Comment (TheElectronWill) [reply composer.json]: Ah, I missed it. Ok then

**Pull request merged**